main.go

```diff
@@ -143,6 +167,9 @@ func checkArgs(_ *corev2.Event) error {
 	if len(config.ToEmail) == 0 {
 		return errors.New("missing destination email address")
 	}
+	if config.Insecure && config.LoginAuth {
+		return fmt.Errorf("--insecure (-i) and --loginauth (-l) flags are mutually exclusive")
```

Comment:
This line needs to be updated with the new flag. Personally `enableLoginAuth` reads a bit better, but if you prefer `useLoginAuth` I'm good with that as well 😄

Reply:
Completed on your suggestion.